src/VisualStudio/Core/Def/Implementation/ProjectSystem/Extensions/ProjectExtensions.cs

```diff
@@ -57,7 +57,28 @@ public static ProjectItem FindItem(this EnvDTE.Project project, string itemName,
 
         public static ProjectItem FindItemByPath(this EnvDTE.Project project, string itemFilePath, StringComparer comparer)
```

Comment:
@jasonmalinowski I've made this recursive now. This was also used by the `CPSCodeModelInstanceFactory.GetProjectItem(string filePath)`. I feel like that one probably wanted to be recursive too, but I'm not sure. What do you think?

Reply:
That's probably what I expected too.